packages/http-client-python/generator/pygen/codegen/models/operation.py

```diff
@@ -314,6 +314,10 @@ def get_request_builder_import(
             )
         return file_import
 
+    @property
+    def need_deserialize(self) -> bool:
+        return any(r.type and not isinstance(r.type, BinaryIteratorType) for r in self.responses)
+
```

Comment:
the type judge is weird. what kind of response type does not need serialization?

Reply:
with offline sync. i got the idea. but need optimization in the future.